moveit_setup_assistant/resources/default_config/sensors_3d.yaml

```diff
@@ -0,0 +1,20 @@
+# This YAML configuration file holds the default values used for configuring 3D sensors
```

Comment:
Add comment:

     TODO: values may not be ideal defaults, original source unclear